### DISCUSSION TITLE
Feel free to ask if need help getting started with Quibbler !

### DISCUSSION THREAD

**Discussion by rkishony** [Technion-Kishony-lab/quibbler#555] [Q&A]
if you get stuck getting started with Quibbler - let us know. It will help clarify as other people are likely also stuck on similar issues.

**Comment by leedel2** [comment]
Hi. Very nice package. I'm starting to work with quibbler and after running the first example block, the notebook gets stuck after each block running, and to keep running it I need to go back to the first block output, and hit one of the options. Can you help debug?
